### PR TITLE
AvatarService fully tested

### PR DESCRIPTION
**Pull request by marcinwyszynski**
@codequest-eu/developers PTAL

### CODE REVIEW

**Comment by BartlomiejSkwira** [comment]
is the doge image copyrighted?

File: golang/app/models/mocks/mock_storage_manager.go [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package mocks
+
+import (
+	"io"
+	"net/url"
+	"time"
+
```
Comment (BartlomiejSkwira) [inline golang/app/models/mocks/mock_storage_manager.go]: Unnecessary space? :P

File: golang/app/services/avatar_service_test.go [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+	"time"
+
+	"github.com/satori/go.uuid"
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/suite"
+	"golang.org/x/net/context"
+
+	"github.com/codequest-eu/startupinabox/golang/app/models/mocks"
+	"github.com/codequest-eu/startupinabox/golang/app/models"
+	siab "github.com/codequest-eu/startupinabox/golang/stubs"
```
Comment (BartlomiejSkwira) [inline golang/app/services/avatar_service_test.go]: siab?

File: golang/app/models/mocks/mock_storage_manager.go (hunk 1 above)
Comment (marcinwyszynski) [inline golang/app/models/mocks/mock_storage_manager.go]: No, grouped imports.

File: golang/app/services/avatar_service_test.go (hunk 2 above)
Comment (marcinwyszynski) [inline golang/app/services/avatar_service_test.go]: **S**tartup **I**n **A** **B**ox

Comment (BartlomiejSkwira) [inline golang/app/services/avatar_service_test.go]: 👍

File: golang/app/models/mocks/mock_storage_manager.go (hunk 1 above)
Comment (BartlomiejSkwira) [inline golang/app/models/mocks/mock_storage_manager.go]: Ok, thanks

**Pull request merged**